BookingsApi/BookingsApi.DAL/Migrations/20210813101442_AddInterpreterRoleForGRCEJ.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddInterpreterRoleForGRCEJ : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                table: nameof(HearingRole),
+                columns: new[] { "Id", "Name", "UserRoleId", "CaseRoleId", },
+                values: new object[,]
+                {
+                    {756, "Interpreter", 5, 203},
```

Comment:
Can you use this dictionary  to map the user role ID. See Prasannas change here https://github.com/hmcts/vh-bookings-api/pull/312/files